playwright/core/defaultDelay.ts

```diff
@@ -0,0 +1,7 @@
+import {DEFAULT_MOUNT_TEST_DELAY} from './constants';
+import type {PlaywrightFixture} from './types';
+
+export const defaultDelayFixture: PlaywrightFixture<() => Promise<void>> = async ({page}, use) => {
+    const defaultDelay = async () => await page.waitForTimeout(DEFAULT_MOUNT_TEST_DELAY);
```

Comment:
can we use smth like this?
```
await page.waitForLoadState('load');
```
waiting for `load` event can be more robust technique for a waiting for a page to load